src/Raven.Server/Web/Studio/StudioTasksHandler.cs

```diff
@@ -71,25 +77,65 @@ public async Task FullDataDirectory()
             await info.UpdateDirectoryResult(databaseName: null, error: error);
         }
 
-        [RavenAction("/admin/studio-tasks/folder-path-options", "GET", AuthorizationStatus.Operator)]
-        public Task GetFolderPathOptions()
+        [RavenAction("/admin/studio-tasks/folder-path-options", "POST", AuthorizationStatus.Operator)]
+        public async Task GetFolderPathOptions()
         {
-            var path = GetStringQueryString("path", required: false);
-            var isBackupFolder = GetBoolValueQueryString("backupFolder", required: false) ?? false;
-
-            var folderPathOptions = FolderPath.GetOptions(path, isBackupFolder, ServerStore.Configuration);
+            var connectionType = GetBackupConnectionType();
+            
             using (ServerStore.ContextPool.AllocateOperationContext(out JsonOperationContext context))
-            using (var writer = new BlittableJsonTextWriter(context, ResponseBodyStream()))
             {
-                context.Write(writer, new DynamicJsonValue
+                FolderPathOptions folderPathOptions;
+                switch (connectionType)
                 {
-                    [nameof(FolderPathOptions.List)] = TypeConverter.ToBlittableSupportedType(folderPathOptions.List)
-                });
-            }
+                    case PeriodicBackupConnectionType.Local:
+                        var isBackupFolder = GetBoolValueQueryString("backupFolder", required: false) ?? false;
+                        var path = GetStringQueryString("path", required: false);
+                        folderPathOptions = FolderPath.GetOptions(path, isBackupFolder, ServerStore.Configuration);
+                        
+                        break;
+                    case PeriodicBackupConnectionType.S3:
+                        var json = context.ReadForMemory(RequestBodyStream(), "studio-tasks/format");
```

Comment:
I wonder if we should comment this out, as we won't support listing in s3 buckets for now, as we only support cloud?

Reply:
@grisha-kotler